### ISSUE TITLE
Difficulties in unlearning contrast bias T1/T2 in IXI brain dataset using VGG based network 

### ISSUE THREAD

**Issue by surajsrivathsa** [nkdinsdale/Unlearning_for_MRI_harmonisation#2]
I am trying to unlearn contrast information ie: T1/T2 from IXI Brain dataset using VGG based network.
The contrast bias seems to be much stronger than scanner/sex/age bias.
Hence the network doesn't unlearn the contrast information.

I tried to tinker around learning rates and loss weightage for contrast, but it seems like either network fails to unlearn or becomes too unstable.

Could you please let me know any encounters or experiments done on removing strong biases from the network.

**Comment by nkdinsdale** [comment]
Hi, I havent tried removing contrast information but I think it could be hard to remove because the intensity information will be driving the age prediction and you would need to remove that information. While there is other information such as ventricle size that could give a good prediction for age, the intensity information is likely to be what drives the prediction. 

If I were to try removing the contrast information I'd pretrain the model to convergence for the main task and then remove it really slowly. Id also try and have a very large batch size.

**Comment by surajsrivathsa** [comment]
Thank you for the suggestions. I will try them out.
